.github/workflows/deploy.yml

```diff
@@ -61,22 +56,32 @@ jobs:
         if: github.ref == 'refs/heads/develop'
         uses: codfish/semantic-release-action@v1
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GH_ACTION_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
       - name: Deploy on master branch
         if: github.ref == 'refs/heads/master'
         uses: codfish/semantic-release-action@v1
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GH_ACTION_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
-      - name: Merge to master branch
-        if: github.ref == 'refs/heads/master'
+  sync_master_to_development_branch :
+    if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/develop'
```

Comment:
Why both `master` and `develop` here?

Reply:
We want to make sure that anytime we push into develop also, we keep this synchronization with master branch.
Even though most of the time develop branch will be ahead of master, it is still possible to have some pushes on master that aren't merged into develop (for example if an admin commits directly on master)

Reply:
So I think what you're saying is that this pipeline isn't run if an admin commits directly on `master`? I think that's the only way that commits could end up on master and not be synced back to develop?